configs/templates/ufs-srw-public-v2/spack.yaml

```diff
@@ -0,0 +1,47 @@
+spack:
+  config:
+    install_tree:
+      root: $env/install
+  modules:
+    default:
+      roots:
+        lmod: $env/install/modulefiles
+        tcl: $env/install/modulefiles
+      lmod:
+        projections:
+          parallelio: 'parallelio/2.5.2'
```

Comment:
PIO has a messed up tag with an extra underscore, and the module is installed as 2_5_2.